### ISSUE TITLE
[New Feature]: Job Completion Task

### ISSUE THREAD

**Issue by LucaCinquini** [unity-sds/unity-sps-prototype#187]
Create a task that, when a job execution is completed, will send a message to an SNS topic with all the relevant information.
The functionality needs to be packaged as a Docker container and invoked as a CWL workflow.

**Comment by LucaCinquini** [comment]
The same Docker container that is mused to send the job creation message can also be used to update its status. The wrapper workflow calls them both before and after the CHIRP execution.

**Comment by rtapella** [comment]
Does "completion" include failure/error? Is it the same topic?

**Comment by LucaCinquini** [comment]
Yes, it should definitely include notification to the database in case of error.
